comms/Dockerfile

```diff
@@ -5,11 +5,9 @@ FROM golang:latest
 RUN apt update && apt install -y ffmpeg 
 
 # install in-container tools: dbmate, nats cli
-RUN cd /tmp && \
-  curl -fsSL -o /usr/local/bin/dbmate https://github.com/amacneil/dbmate/releases/latest/download/dbmate-linux-amd64 && \
-  chmod +x /usr/local/bin/dbmate && \
-  curl -fsSL -o nats.deb https://github.com/nats-io/natscli/releases/download/v0.0.35/nats-0.0.35-amd64.deb &&\
-  apt install ./nats.deb
+RUN export GO111MODULE=on && \
+  go install github.com/nats-io/natscli/nats@latest && \
+  go install github.com/amacneil/dbmate@latest
```

Comment:
This is probably the best way to do this if it works, if not the bash if is fine